docs/docs/guides/migrate/users.md

```diff
@@ -0,0 +1,231 @@
+  "idps": [
+    {
+      "configId": "124425861423228496",
+      "externalUserId": "roadrunner@mailonline.com",
+      "displayName": "name"
+    }
+  ]
+}
+```
+
+## Bulk import
+
+For bulk import use the [import endpoint](https://zitadel.com/docs/apis/admin/admin-service-import-data) on the admin API:
+
+```json
+{
+  "timeout": "10m",
+  "data_orgs": {
+    "orgs": [
+      {
+        "orgId": "104133391254874632",
+        "org": {
+          "name": "TestFabi"
+        },
+        "humanUsers": [
+          {
+            "userId": "104133391271651848",
+            "user": {
+              "userName": "fabienne",
+              "profile": {
+                "firstName": "Fabienne",
+                "lastName": "Gerschwiler",
+                "displayName": "Fabienne Gerschwiler",
+                "preferredLanguage": "und"
+              },
+              "email": {
+                "email": "fabienne@caos.ch",
+                "isEmailVerified": true
+              },
+              "hashedPassword": {
+                "value": "$2a$14$aPbwhMVJSVrRRW2NoM/5.esSJO6o/EIGzGxWiM5SAEZlGqCsr9DAK",
+                "algorithm": "bcrypt"
+              }
+            }
+          },
+          {
+            "userId": "120080115081209416",
+            "user": {
+              "userName": "testuser",
+              "profile": {
+                "firstName": "Test",
+                "lastName": "User",
+                "displayName": "Test User",
+                "preferredLanguage": "und"
+              },
+              "email": {
+                "email": "fabienne@caos.ch",
+                "isEmailVerified": true
+              },
+              "hashedPassword": {
+                "value": "$2a$14$785Fcdbpo9rn5L7E21nIAOJvGCPgWFrZhIAIfDonYXzWuZIKRAQkO",
+                "algorithm": "bcrypt"
+              }
+            }
+          },
+          {
+            "userId": "145195347319252359",
+            "user": {
+              "userName": "test1",
+              "profile": {
+                "firstName": "Fabienne",
+                "lastName": "Gerschwiler",
+                "displayName": "Fabienne Gerschwiler",
+                "preferredLanguage": "und"
+              },
+              "email": {
+                "email": "fabienne@gmail.com"
+              }
+            }
+          }
+        ]
+      }
+    ]
+  }
+}
+```
+
+:::info
+We will improve the bulk import interface for users in the future.
+You can show your interest or join the discussion on [this issue](https://github.com/zitadel/zitadel/issues/5524).
+:::
+
+## Migrate secrets
+
+Besides user data you need to migrate secrets, such as password hashes, OTP seeds, and public keys for passkeys (FIDO2).
+
+### Passwords
+
+Passwords are stored only as hash.
+You can transfer the hashes as long as ZITADEL [supports the same hash algorithm](/docs/concepts/architecture/secrets#hashed-secrets).
+Password change on the next sign-in can be enforced.
+
+```json
```

Comment:
Is this a part out of the bulk import? Thats not really clear to me. 
Also in the following parts OTP, etc.

Reply:
Yes that's part of the bulk import. I wanted to show the relevant part of the data structure.
I don't have a better idea on how to reference the attributes. Is there a way to point to the API docs?
Should we leave it out?

Reply:
MAybe we can just add a note above that these are snippets out of the bulk import to make it clear?

Reply:
Makes sense, yes. I've added a short text before the snippets to make it clear that these are extracts from the section above. Thanks!